submissions/VikTorNik/a-tiny-JS-world--OOP/index.js

```diff
@@ -0,0 +1,95 @@
+import { print } from './js/lib.js';
+
+class BasePersonage {
```

Comment:
WHy not just `Personage`? You will never know if a class you design wouldn't become a base class for a new class.